### ISSUE TITLE
 save-tabs-to-bookmark-folder_review and suggestion_done!

### ISSUE THREAD

**Issue by ingr8** [igorlogius/save-tabs-to-bookmark-folder#7]
Hello! Thank you very much for prompt action on my review and suggestion on your Firefox add-on Save Tabs to Bookmark Folder.
I tried out the new option of "backing up Selected tabs" and it works fine. But there is a very slight problem. It's a very minor problem and I don't think it's a feature - I think it's a bug.
When I have two or more windows open and I go to save SELECTED tabs in the first window, the tabs of the first window are backed up, but so are the single, active individual tabs in the other windows (but only one in each of the background windows, the last one with the focus).
So if I have four windows open and in the first window I choose to save SELECTED tabs, the saved list will include all tabs from the first window and the active tab from each of the other three windows. It's not a big inconvenience, but I do have to open the bookmark manager and delete the unwanted tabs.

If I have four open windows and in the first window I choose to save ALL tabs, the tabs are all saved no problem. And there are no other tabs saved from the other three windows. They are left alone. So the problem described above is only a problem when dealing with "save SELECTED tabs" and not with "save ALL tabs"..

I don't know if it's just me or the add-on. Please check it out. Thank you for your add-on and for your help in this.

**Comment by igorlogius** [comment]
You are right, i initally thought it would make sense to allow the user to save tabs accross windows ... but i suspect with there always being an selected tab in a window ... that doesnt make much sense. I'll change the behaviour of the addon, to only include the tabs from the currently active window.  Thanks for the feedback.

**Comment by igorlogius** [comment]
I'll push the version with this fix to AMO, but i will very soon after replace it, so if you want to keep it you can, but i wont be updating it. 
Sorry.  (see. #6)

**Comment by ingr8** [comment]
Thank you for your quick reply.
Are you going to abandon this add-on? Are you going to replace it will a different, better version? Will there be one last update to the current 1.3.6? I'm not entirely sure what you mean. If it's not too much trouble could you explain? Thanks.

________________________________

**Comment by igorlogius** [comment]
No i will not abandon the addon itself. 
Only the "Selection Feature" will not be included in versions beyond  1.3.7.
The reason is as i explained in #6 that i think it is unnecessary, since one can already move selected tabs into a new window which allows then to only bookmark those tabs with the existing functionality.
